azure-pipelines.yml

```
@@ -11,7 +11,7 @@ steps:
 - task: DotNetCoreInstaller@0
   displayName: 'Installing .NET Core SDK...'
   inputs:
-    version: 3.0.100
+    version: 3.1.0-preview2
```

I think you've got the wrong SDK version here @peterblazejewicz. I think it should be `3.1.100-preview2-014569`.
```suggestion
    version: 3.1.100-preview2-014569
```

---

amended, thanks!